tests/run/toplevel-overloads/defs_1.scala

```diff
@@ -0,0 +1,8 @@
+package top
+
+def hello(name: String) = s"hello, $name"
+def hello(x: Int) = x.toString
+
+object O {
+  def hi = hello("Bob")
+}
```

Comment:
Could we also have a top-level `def` and a non-top-level `def` in this file that call `hell(Boolean)`?

Reply:
Hum, I was referring to calling the `def hello(b: Boolean)` that was already defined in `moredefs_1.scala`, to test that overloading resolution can find overloads in other files.

And right now I believe the code *shouldn't* compile, because it shouldn't be able to pick between the `def hello(Boolean)`s that are defined in this file and in `moredefs_1.scala`.

Reply:
Indeed it did not compile. The test is now as you propose.